File: src/iniciante/Bee1018.java
package iniciante;

import java.util.Scanner;

public class Bee1018 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int n = input.nextInt();
        int quantidadeNotas;
        int divisores[] = {100, 50, 20, 10, 5, 2, 1};

        System.out.println(n);
        for (int i = 0; i < 7; i++) {
            quantidadeNotas = (n/divisores[i]);
            n %= divisores[i];
            System.out.println(String.format("%d nota(s) de R$ %d,00", quantidadeNotas, divisores[i]));
        }

    }
}

File: beginner/bee1018.py
def banknotes(value):
    while value != 0:
        print(value)
        print(f'{value // 100} nota(s) de R$ 100,00')
        value = value % 100
        print(f'{value // 50} nota(s) de R$ 50,00')
        value = value % 50
        print(f'{value // 20} nota(s) de R$ 20,00')
        value = value % 20
        print(f'{value // 10} nota(s) de R$ 10,00')
        value = value % 10
        print(f'{value // 5} nota(s) de R$ 5,00')
        value = value % 5
        print(f'{value // 2} nota(s) de R$ 2,00')
        value = value % 2
        print(f'{value // 1} nota(s) de R$ 1,00')
        value = value % 1

value = int(input())
banknotes(value)
